modules/ssh/init.zsh

```diff
@@ -6,10 +6,15 @@
 #
 
 # Return if requirements are not found.
-if [[ "$OSTYPE" == darwin* ]] || (( ! $+commands[ssh-agent] )); then
-  return 1
+if [[ "$OSTYPE" == darwin* ]]; then
+    if [[ "$(which ssh-agent)" == "/usr/bin/ssh-agent" ]]; then
```

Comment:
You do not need `which`. Zsh has the `$commands` array. Furthermore, you're making assumptions about where `ssh-agent` is installed, in this case `/usr/bin`. The logic here needs to change.